### PR TITLE
example: Revise meson script in example

### PR DESCRIPTION
**Pull request by JSUYA**

### CODE REVIEW

File: example/meson.build [hunk 1]
```diff
@@ -1,67 +1,20 @@
...
-
-demo4_source_file = [
-   'user_interaction_rollinout.cpp'
+source_file = [
+   'rive_viewer.cpp',
+   'runtime_property_change.cpp',
+   'animation_mixing.cpp',
+   'user_interaction_rollinout.cpp',
+   'user_interaction_follow_cursor.cpp',
+   'user_interaction_animation_speed.cpp',
```
Comment (taehyub) [inline example/meson.build:9]: please update user_interaction_touch.cpp

**Comment by JSUYA** [comment]
rebased patch

**Review by taehyub — APPROVED** [review]
looks good to me

File: example/meson.build (hunk 1 above)
Comment (JSUYA) [reply example/meson.build:9]: added thanks

**Pull request merged**